addons/statemachine/CBA_FSMEditor.cfg

```diff
@@ -0,0 +1,103 @@
+/* ----------------------------------------------------------------------------
+CBA Statemachine Config for use with FSM Editor
+
+Description:
+    This is a config that allows you to create and edit CBA Statemachines using
+    the Arma Tools FSM editor, for increased readability and expandability.
+	
+	It isn't by any means perfect, but it should be useable
+    
+Usage:
+    1. Select this config using FsmAttributes > Compile Config
+    2. Change FSM attributes under FsmAttributes > Attributes from Compile Config
+        2.1 'list' parameter is parsed as list = QUOTE(x);
+        2.2 'skipNull' parameter is parsed as skipNull = x;
+    3. States
+        3.1 'initCode' is parsed as is, and may contain onState = "", 
+            onStateEntered = "" and/or onStateLeaving = ""
+    4. Transitions
+        4.1 'targetState' does not need to be entered, and simply follows the
+            transition link
+        4.2 'condition' is parsed as condition = QUOTE(x)
+        4.3 'action' is parsed as is, and may contain onTransition = "" 
+            and/or events[] = {}
+			
+Dependencies:
+	In order to properly function, script_macros_common need to be included
+
+Author:
+    Freddo 2019-12-10
+---------------------------------------------------------------------------- */
+
+class Attributes
+{
```

Comment:
Does it have to have open curly bracket after newline? Goes against the rest of the code base.